app/assets/javascripts/utils/course_utils.js

```diff
@@ -117,22 +111,22 @@ const CourseUtils = class {
     return `https://${language}.${project}.org/wiki/${underscoredTitle}`;
   }
 
-  formattedArticleTitle(language, project, articleTitle) {
+  formattedArticleTitle(language, project, title, home) {
     let languagePrefix = '';
-    if (language === undefined || language === 'en') {
+    if (!home || language === home.language || language === null || language === undefined || language === 'en') {
```

Comment:
We can probably do away with the special-casing of 'en' here. Also, see if you can simplify this conditional any further.